integrations/ramp-sandbox/nango.yaml

```diff
@@ -0,0 +1,64 @@
+integrations:
+    ramp-sandbox:
+        syncs:
+            users:
+                runs: every day
+                description: Fetches a list of users from Ramp
+                output: User
+                sync_type: full
+                track_deletes: true
+                endpoint:
+                    method: GET
+                    path: /users
+                    group: Users
+                scopes:
+                    - users:read
+
+        actions:
+            create-user:
+                description: Creates a user in Ramp
+                output: User
+                endpoint:
+                    method: POST
+                    path: /users
+                    group: Users
+                input: RampCreateUser
+                scopes:
+                    - users:write
+
+            delete-user:
```

Comment:
This is actually deactivating a user so the action should be called `disable-user`